### DISCUSSION TITLE
Endless Possibilities

### DISCUSSION THREAD

**Discussion by shucks18** [smartcontractkit/full-blockchain-solidity-course-py#1278] [General]
This course has gotten me from zero to knowing so much about how the blockchain works and how to work with it! A big shoutout to @PatrickAlphaC  for putting this amazing course together , and all the help from you fellow peeps on github especially @cromewar .

Im planning to continue to work on some projects/ideas with brownie, but I know I will have to inevitably learn some JS one day because most of the people in web3 are JS devs :( but it's alright, whatever it takes to learn :p 

Also if any of you are down to work on some projects in the future just reply to this thread and we'll talk from there!

**Comment by n4n0b1t3** [comment]
Hi @shucks18, I am definitely looking for some collaboration projects. Currently, I am still in lesson 7 but this is because I am a stickler when it comes to understanding what I am doing. You will like JS, it's really powerful and an art. I liked the book Eloquent in JavaScript by Haverbeke. Maybe not the best to get started though.

**Reply by shucks18** [reply]
All the best with the course man you'll learn a lot more in the coming lessons! I'll definitely check out the book thanks :) , Also do you have any other beginner resources for JS

**Reply by n4n0b1t3** [reply]
My son just did a Udemy course and was quite happy. Search for something for vanilla JavaScript and the basics, because if you start too early with frameworks like NodeJS, React, and the likes you might be overwhelmed. Some frameworks introduce some patterns one might easily misinterpret as the vanilla syntax, this IMHO was the fact with jQuery.
This one here has all the basics: https://www.udemy.com/course/javascript-beginnerscourse/

I probably would do this one here, because it just takes 5 hours: https://www.udemy.com/course/learn-to-code-in-javascript/

**Reply by cromewar** [reply]
Nice info @n4n0b1t3 
@shucks18 if you are interested on participate on the next big hackathon coming, we could talk about creating a team.
Cheers.

**Reply by shucks18** [reply]
@n4n0b1t3 Appreciate the advice i'll definitely check out the links :) @cromewar oo which hackathon is this?

**Reply by n4n0b1t3** [reply]
just registered for the upcoming hackathon \o/ 
[Chainlink Spring 2022 Hackathon](https://chain.link/hackathon?utm_medium=team&utm_source=dev-advocate&utm_campaign=spring-2022-hackathon&utm_content=1)
I have to consult with you guys on how to plan this. Don't have so much vacation to burn... @cromewar Do you know how much time one needs to invest so that it makes any sense to participate?

**Reply by cromewar** [reply]
The hackathon duration is less or more 1 month, so may be 1 or two hours a day.

**Reply by n4n0b1t3** [reply]
@cromewar OK, cool I think I can manage that.

**Reply by shucks18** [reply]
@cromewar yeah im down!

**Reply by cromewar** [reply]
Nice let's keep in touch then!!

**Reply by PatrickAlphaC** [reply]
This group here would be an INSANE team :)

**Reply by cromewar** [reply]
@PatrickAlphaC This hackathon is going to be huge

**Reply by shucks18** [reply]
hey @cromewar @n4n0b1t3  where can I connect with you guys? telegram, discord, twitter?

**Reply by n4n0b1t3** [reply]
Go to my GitHub Profile, there you can find my LinkedIn link

**Reply by cromewar** [reply]
"@cromewar" on Twitter ;)
